soh/soh/Enhancements/randomizer/3drando/shops.cpp

```diff
@@ -338,138 +338,138 @@ void InitTrickNames() {
      Text{"Bottle with Heart Potion", "Flacon de potion de soin", "Botella de poción de salud"}};
   trickNameTable[0x8D] = {
      Text{"Bottle with Green Chu Jelly", "Flacon de gelée Chuchu verte", "Jugo de Chuchu Verde"},
-     Text{"Bottle with Medicine of Magic", "Flacon d'élixir vert", "Botella de medicina mágica"},
+     Text{"Bottle with Lamp Oil", "Flacon de Huile à lanterne", "Botella de Aceite de candil "},
      Text{"Bottle with Stamina Potion", "Flacon d'Endurol", "Botella de elixir vigorizante"}};
   trickNameTable[0x8E] = {
      Text{"Bottle with Blue Chu Jelly", "Flacon de gelée Chuchu bleue", "Jugo de Chuchu Azul"},
      Text{"Bottle with Water of Life", "Flacon d'élixir bleu", "Botella de agua de la vida"},
      Text{"Bottle with Air Potion", "Flacon de potion d'oxygène", "Botella de oxígeno"}};
   trickNameTable[0x8F] = {
      Text{"Bottle with Forest Firefly", "Flacon avec une luciole", "Luciérnaga del bosque"},
-     Text{"Bottle with Faerie", "Flacon de poudre féérique", "Gran Hada embotellada"},
+     Text{"Bottle with Deku Princess", "Flacon avec Deku Princess", "Botella con Deku Princess"},
      Text{"Bottle with Stray Fairy", "Flacon avec une fée perdue", "Hada perdida en una botella"}};
   trickNameTable[0x90] = {
-     Text{"Bottle with Small Jabu-Jabu", "Flacon avec mini Jabu-Jabu", "Lord Chapu-Chapu embotellado"},
+     Text{"Bottle with Reekfish", "Flacon avec Reekfish", "Reekfish embotellada"},
      Text{"Bottle with Hyrule Bass", "Flacon avec perche d'Hyrule", "Locha de Hyrule embotellada"},
      Text{"Bottle with Hyrule Loach", "Flacon avec loche d'Hyrule", "Perca de Términa embotellada"}};
   trickNameTable[0x91] = {
-     Text{"Bottle with Will-O-Wisp", "Flacon avec feu follet", "Botella de llama azul"},
-     Text{"Bottle with Ancient Flame", "Flacon de flamme ancienne", "Botella de fuego ancestral"},
+     Text{"Bottle with a Blue Candle", "Flacon avec une bougie bleue", "Botella con una vela azul"},
+     Text{"Bottle with Red Ice", "Flacon de Glace Rouge", "Botella de Hielo rojo"},
      Text{"Bottle with Nayru's Flame", "Flacon de flamme de Nayru", "Botella de llamas de Nayru"}};
   trickNameTable[0x92] = {
-     Text{"Bottle with Baby Tektites", "Flacon de bébé Araknon", "Tektites en una botella"},
+     Text{"Bottle with A Beetle", "Flacon avec un scarabée", "Botella con un escarabajo"},
      Text{"Bottle with Lanayru Ants", "Flacon de fourmis de Lanelle", "Celestarabajo embotellado"},
-     Text{"Bottle with Insects", "Flacon de bibittes", "Saltabosques embotellados"}};
+     Text{"Bottle with a Golden Bee", "Flacon avec une abeille dorée", "Botella con una abeja dorada"}};
   trickNameTable[0x94] = {
      Text{"Bottle with Ghini", "Flacon avec Ghini", "Ghini en una botella"},
-     Text{"Bottle with Imp Poe", "Flacon avec Spectre", "Espectro en una botella"},
+     Text{"Bottle with Reapling", "Flacon avec Âme Damnée", "Reapling en una botella"},
```

Comment:
The old one feels better at least in french

Reply:
Quite possibly, I warned people at the start i had no idea what I was doing with the other languages.